aries-site/src/examples/components/layer/LayerExample.js

```diff
@@ -9,33 +9,50 @@ import {
   useConfirmation,
 } from './components';
 
-export const LayerExample = () => (
-  <ConfirmationProvider>
-    <ConfirmationContext.Consumer>
-      {({ showLayer, showConfirmation, setShowLayer }) => (
-        <>
-          <Box align="center">
-            <Button
-              label="Show me the layer"
-              onClick={() => setShowLayer(true)}
-              primary
-            />
-          </Box>
-          {showLayer ? <AddApplication /> : null}
-          {showConfirmation ? <DoubleConfirmation title="application" /> : null}
-        </>
-      )}
-    </ConfirmationContext.Consumer>
-  </ConfirmationProvider>
-);
+export const LayerExample = () => {
+  const announce = useContext(AnnounceContext);
+
```

Comment:
Can we do something like:
```suggestion
const modalTitle = "Add application modal";
```

Then in each of the announces:
```
announce(`${modalTitle} opened.`, 'polite');

....

announce(`${modalTitle} closed.`, 'polite');
```

This way the messages are synched / aligned.